wp-content/themes/core/pcss/content/panel/_gallery.pcss

```diff
@@ -6,38 +6,110 @@
 
 .site-panel--gallery {
 
-	.c-slider__slide {
-		aspect-ratio: '16:10';
+	.g-col {
+		padding-bottom: var(--spacer-2);
+
+		@media (--viewport-medium) {
+			padding-bottom: var(--spacer-4);
+		}
+	}
+}
+
+.site-panel--gallery__content {
+	max-width: 848px;
+	width: 100%;
+
+	.site-panel--gallery--content-inline & {
+		max-width: 512px;
+		padding: 0;
 	}
+}
 
-	.c-slider__main--crop {
+.c-btn.site-panel--gallery__btn {
 
-		.tribe-image {
-			position: absolute;
-			width: 100%;
-			height: 100%;
-			top: 0;
-			left: 0;
-			background-position: 50% 50%;
-			background-size: cover;
+	.t-content & {
+		margin-top: 20px;
+	}
+}
+
+.site-panel--gallery__slider {
```

Comment:
I _think_ we should turn a lot of the themery that is part of skinning this slider into a slider theme that can be used anywhere through the use of a c-slider--variation parent class. Let's chat.